uploadNeuroDB/tarchive_validation.pl

```diff
@@ -291,38 +292,13 @@ =head2 Methods
 ################################################################
 ################################################################
 my $subjectIDsref = $utility->determineSubjectID(
-    $scannerID, \%tarchiveInfo, 1, $upload_id
-);
-
-################################################################
-################################################################
-## Optionally create candidates as needed Standardize sex    ###
-## (DICOM uses M/F, DB uses Male/Female) #######################
-################################################################
-################################################################
-$utility->CreateMRICandidates(
-    $subjectIDsref, $sex, \%tarchiveInfo, $User, $centerID, $upload_id
+    $scannerID, \%tarchiveInfo, 1, $upload_id, $User, $centerID
 );
-
-################################################################
-################################################################
-## Check the CandID/PSCID Match It's possible that the CandID ## 
-## exists, but doesn't match the PSCID. This will fail further #
-## down silently, so we explicitly check that the data is ######
-## correct here. ###############################################
-################################################################
-################################################################
-my $CandMismatchError= $utility->validateCandidate($subjectIDsref);
-if (defined $CandMismatchError) {
-    print "$CandMismatchError \n";
+if (defined $subjectIDsref->{'CandMismatchError'}) {
+    print "$subjectIDsref->{'CandMismatchError'} \n";
     ##Note that the script will not exit, so that further down
     ##it can be inserted per minc into the MRICandidateErrors
 }
-################################################################
-############ Get the SessionID #################################
-################################################################
-my ($sessionID) =
-    $utility->setMRISession($subjectIDsref, \%tarchiveInfo, $upload_id);
```

Comment:
remove this as `$sessionID` is not used here and `setMRISession` is calling `MRI::getSessionID` which would be failing when `CandID` is not valid.